output.tf

```diff
@@ -21,3 +21,7 @@ output "user_name" {
 output "name" {
   value = azurerm_postgresql_server.postgres-paas.name
 }
+
+output "resource_group_name" {
+  value = azurerm_template_deployment.postgres-paas.resource_group_name
```

Comment:
this branch doesn't use ARM templates: https://github.com/hmcts/cnp-module-postgres/blob/postgresql_tf/main.tf#L32

Reply:
right, I just cherry-pick'd
thats what I get for being lazy, fixed